Restructure our XR pages and introduce additional information (#12118)
godotengine/godot-docs: tutorials/xr/setting_up_xr.rst

        }
    }

.. info::

>     There is no restriction to where this code is executed from. It is common to add this script to the :ref:`XROrigin3D <class_xrorigin3d>` node or as a :ref:`Node3D <class_node3d>` child node of the root node.

Reviewer: ```suggestion
    There is no restriction to where this code is executed from. It is common to add this script to the :ref:`XROrigin3D <class_xrorigin3d>` node or as a :ref:`Node3D <class_node3d>` child of the root node.
```

>     The OpenXR interface is unique in that we have to start it before your project loads, hence ``is_initialized`` is checked here. Most interfaces require a call to their ``initialize`` function instead.

Reviewer: ```suggestion
    The OpenXR interface is unique in that we have to start it before our project loads, hence ``is_initialized`` is checked here. Most interfaces require a call to their ``initialize`` function instead.
```

Author: Hmmm, our sounds like Godot's, not the developers. Maybe `the` just to limit confusion?

Reviewer: But it mixes "we" with "your" which is confusing, but "the" works as well

Author: Yes, We as in Godot, need to start OpenXR (as in, this is built in behavior), but you, as in the developer, need to therefor check if it is initialized instead of initializing yourself. But I think the works better anyway so I changed it to that :)

Reviewer: Got it, it isn't very clear that the "we" isn't the reader here

    If you wish to support multiple XR interfaces, say release a game both targeting OpenXR hardware and deploy over WebXR, you can check one after the other until a functioning interface is found.

.. warning::

>     As OpenXR outputs the rendering result to an HMD, which often runs at a higher framerate than the monitor, Godots V-Sync settings are ignored and V-sync will always be disabled.

Reviewer: ```suggestion
    As OpenXR outputs the rendering result to an HMD, which often runs at a higher framerate than the monitor, Godot's V-Sync settings are ignored and V-sync will always be disabled.
```

>     Instead, OpenXR perform it's own frame timing to ensure a consistant framerate.

Reviewer: ```suggestion
    Instead, OpenXR perform its own frame timing to ensure a consistent framerate.
```
